cves/CVE-2013-0863.yml

```diff
@@ -355,11 +365,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
-nickname_instructions: |
+CWE: 119
+CWE_note: Improper Restriction of Operations within the Bounds of a memory buffer.
+nickname_instructions: 
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: Thats not a pointer thats my wife!
```

Comment:
Another solid nickname. Nice work!

Reply:
+2 upvote

Fairly basic mistake, but had lots of impact.

Reply:
Did this review second so some of my comments refer to the other PR. the nickname is over 30 characters with spaces, not sure if that counts but wanted to bring it up. +1 upvote